_data/projects/search/releases/7.1/7.1.0.Alpha1.yml

```diff
@@ -0,0 +1,27 @@
+date: 2023-09-05
+announcement_url: https://in.relation.to/2023/11/30/hibernate-search-7-1-0-Alpha1
+summary: >-
+  Introduction of vector search support in the Lucene backend,
+  other bugfixes and improvements
+maven:
+  artifacts:
+    - artifact_id: hibernate-search-bom
+      summary: Hibernate Search BOM
+    - artifact_id: hibernate-search-mapper-orm
+      summary: Hibernate ORM mapper
+    - artifact_id: hibernate-search-mapper-orm-coordination-outbox-polling
+      summary: "\"outbox-polling\" coordination strategy for the Hibernate ORM mapper"
+    - artifact_id: hibernate-search-mapper-pojo-standalone
+      summary: Standalone POJO mapper
+    - artifact_id: hibernate-search-backend-lucene
+      summary: Lucene backend
+    - artifact_id: hibernate-search-backend-elasticsearch
+      summary: Elasticsearch/OpenSearch backend
+    - artifact_id: hibernate-search-backend-elasticsearch-aws
+      summary: Amazon IAM authentication for Elasticsearch/OpenSearch
+    - artifact_id: hibernate-search-mapper-orm-batch-jsr352-core
+      summary: Jakarta Batch mass indexing job for the Hibernate ORM mapper - Core
+    - artifact_id: hibernate-search-mapper-orm-batch-jsr352-jberet
+      summary: Jakarta Batch mass indexing job for the Hibernate ORM mapper - JBeret specifics
+    - artifact_id: hibernate-search-v5migrationhelper-orm
+      summary: Helper for migrating from Hibernate Search 5 to Hibernate Search 6/7 (Hibernate ORM mapper + Lucene backend)
```

Comment:
You don't need this section in every release file; it will default to the one from the series file.

If you see it in some release files, it means the artifacts are slightly different for that release compared to all other releases in the same series. Which is not the case here.

```suggestion
```